tools/godotcpp.py

```diff
@@ -198,6 +198,14 @@ def options(opts, env):
         )
     )
 
+    opts.Add(
+        PathVariable(
+            key="profile",
+            help="Allow specification of customization file other than `custom.py`.",
+            default=""
```

Comment:
```suggestion
            default=None,
            validator=validate_file,
```
My bad didn't notice the other case, might also be best to do:
```suggestion
            default=env.get("profile", None),
            validator=validate_file,
```